tikv-configuration-file.md

```diff
@@ -996,6 +996,23 @@ Configuration items related to `rocksdb.defaultcf`
 + Default value: `"256GB"`
 + Unit: KB|MB|GB
 
+### `enable-compaction-guard`
+
++ Enables or disables the compaction guard, which is an optimization to split SST files at TiKV Region boundaries. This optimization can help reduce compaction I/O and allows TiKV to use larger SST file size (thus less SST files overall) while making sure that TiKV can still efficiently clean up stale data when migrating Regions.
```

Comment:
```suggestion
+ Enables or disables the compaction guard, which is an optimization to split SST files at TiKV Region boundaries. This optimization can help reduce compaction I/O so that TiKV can use larger SST file size (thus less SST files overall) and at the time efficiently clean up stale data when migrating Regions.
```

Reply:
"use larger SST file size" is not the consequence of "reduce compaction I/O"